lib/Dialect/SV/Transforms/HWExportModuleHierarchy.cpp

```diff
@@ -84,9 +84,12 @@ void HWExportModuleHierarchyPass::runOnOperation() {
   Optional<SymbolTable> symbolTable = None;
   bool directoryCreated = false;
 
+  llvm::errs() << "!starting\n";
   for (auto op : mlirModule.getOps<hw::HWModuleOp>()) {
-    if (auto attr = op->getAttrOfType<hw::OutputFileAttr>(
-            "firrtl.moduleHierarchyFile")) {
+    auto attr =  op->getAttrOfType<ArrayAttr>("firrtl.moduleHierarchyFile");
+    if (!attr)
+      continue;
+    for (auto file : attr.getAsRange<hw::OutputFileAttr>()) {
```

Comment:
This change is super lazy - we could cache the file contents instead of regenerating them.  I doubt this really matters though